### PR TITLE
DBZ-6759 - Added new database index connection parameter to Redis Sink Adapter

### PR DESCRIPTION
**Pull request by mfortunat**

### CODE REVIEW

File: debezium-server-redis/src/main/java/io/debezium/server/redis/RedisStreamChangeConsumer.java [hunk 1]
```diff
@@ -95,7 +95,8 @@ else if (MESSAGE_FORMAT_COMPACT.equals(messageFormat)) {
             };
         }
 
-        RedisConnection redisConnection = new RedisConnection(config.getAddress(), config.getUser(), config.getPassword(), config.getConnectionTimeout(),
+        RedisConnection redisConnection = new RedisConnection(config.getAddress(), config.getDbIndex(), config.getUser(), config.getPassword(),
```
Comment (jpechane) [inline debezium-server-redis/src/main/java/io/debezium/server/redis/RedisStreamChangeConsumer.java:98]: @mfortunat The method `getDbIndex` is missing from `RedisStreamChangeConsumerConfig`

**Review by jpechane — CHANGES_REQUESTED** [review]
Thanks for the PR. It seems that not all changes in the codebase were committed. Could you please doublecheck?

**Comment by jpechane** [comment]
@mfortunat I am sorry, I missed https://github.com/debezium/debezium/pull/4751

**Comment by jpechane** [comment]
@mfortunat Applied, thanks a lot! Could you please updated the Redis docs too?

**Comment by mfortunat** [comment]
> @mfortunat Applied, thanks a lot! Could you please updated the Redis docs too?

Thank you @jpechane!
Sure, I'll update the doc as well.

Do I need another Jira ticket for that?

**Pull request merged**